src/components/Reviews.jsx

```diff
@@ -0,0 +1,50 @@
+import { useEffect, useState } from "react"
+function Reviews () {
+    const [reviews, setReviews] = useState([])
+
+    useEffect(() => {
+        fetch(`https://board-games-seo7.onrender.com/api/reviews`)
```

Comment:
I would strongly recommend separating the data fetching into a util folder (with api in it) - you can make a base url here and then add end points on to it depending on the data you are fetching